load-test/results/article/article-detail.md

```
@@ -0,0 +1,33 @@
+- 대상 API: GET /api/articles/{articleId}
+- 테스트 시간: 71초
+- 최대 동시 사용자: 300명
+
+## 테스트 시나리오
+- 0 ~ 30초: 50명으로 증가
+- 30 ~ 60초: 300명으로 증가
+- 60 ~ 70초: 0명으로 감소
+- 10개의 기사 ID 중 랜덤으로 선택하여 요청
+
+## 성능 지표 결과
+| 지표 | 결과 | 기준 | 통과 여부 |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)